### PR TITLE
Changes etcd volume to host path for default

### PR DESCRIPTION
**Pull request by JosephSalisbury**

### CODE REVIEW

File: service/resource/deployment/master_deployment.go [hunk 1]
```diff
@@ -5,14 +5,38 @@ import (
...
+	storageType := key.StorageType(customObject)
+
+	etcdVolume := apiv1.Volume{
+		Name: "etcd-data",
+		VolumeSource: apiv1.VolumeSource{
+			HostPath: &apiv1.HostPathVolumeSource{
+				Path: key.MasterHostPathVolumeDir(key.ClusterID(customObject), key.VMNumber(i)),
+			},
+		},
+	}
```
Comment (xh3b4sd) [inline service/resource/deployment/master_deployment.go]: @kopiczko had something against such defaulting. I am kind of with him. There should be a proper migration for the TPOs. I understand that you want to fix an incident though. So lets go with this and sort it properly afterwards. We only have to find a driver. And no, I will not drive this. <3

**Comment by xh3b4sd** [comment]
Oh, and please fix the CI build before merging.

**Comment by JosephSalisbury** [comment]
@kopiczko thoughts on how to handle the default?

**Comment by kopiczko** [comment]
I think production value here is `PV`. Setting it to `hostPath` would indicate doing something wrong. I generally liked the way it was, i.e. returning error by default.

**Comment by JosephSalisbury** [comment]
This PR is fucked. Brb.

**Pull request closed without merging**